main/src/library/Core/System/Environment.flix

```diff
@@ -0,0 +1,61 @@
+namespace Core/System/Environment {
```

Comment:
Don't forget a license for this file too

Reply:
I've just added it.